Resources/Regression/qml/Correlation.qml

```diff
@@ -72,6 +72,16 @@ Form
 			name: "plotCorrelationMatrix";			label: qsTr("Scatter plots")
 			CheckBox { name: "plotDensities";		label: qsTr("Densities for variables")	}
 			CheckBox { name: "plotStatistics";		label: qsTr("Statistics")				}
+            CheckBox {
+                name: "plotConfidenceIntervals"; label: qsTr("Confidence intervals")
+                childrenOnSameRow: true
+                CIField { name: "plotConfidenceIntervalsInterval"; }
+            }
```

Comment:
It would be nice to change the indenting so that we are as consistent as possible. 
1) In `qml` files, indents should be tabs, not spaces (check the setting of your editor whether it does not replace tab with spaces by default)
2) If a widget does not fit on one line, it's nicer to have every argument on a separate line, and the square brackets should align to each other:

```suggestion
            CheckBox 
            {
                name: "plotConfidenceIntervals"
                label: qsTr("Confidence intervals")
                childrenOnSameRow: true
                CIField { name: "plotConfidenceIntervalsInterval" }
            }
```

Alternatively this would also look ok

```suggestion
            CheckBox 
            {
                name: "plotConfidenceIntervals"; label: qsTr("Confidence intervals"); childrenOnSameRow: true
                CIField { name: "plotConfidenceIntervalsInterval" }
            }
```